enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchLongPatternAcceptanceTest.scala

```diff
@@ -61,7 +61,8 @@ class MatchLongPatternAcceptanceTest extends ExecutionEngineFunSuite with QueryS
 
     // WHEN
     val idpInnerIterations = determineIDPLoopSizes(numberOfPatternRelationships,
-      cypher_idp_solver_table_threshold, maxTableSizes)
+      cypher_idp_solver_table_threshold, maxTableSizes, Map(cypher_idp_solver_duration_threshold -> "10000"))
```

Comment:
these tests actually test that changing these values are affecting the plans produced, but with the default setting when setting the max table sizes to the larger values result in the same plan due to idp running out of time to actually use the increased size